flink-runtime-web/web-dashboard/src/app/pages/job/overview/chart/job-overview-drawer-chart.component.ts

```diff
@@ -44,12 +45,13 @@ import { JobLocalService } from '../../job-local.service';
   imports: [NgIf, NzSelectModule, FormsModule, NgForOf, JobChartComponent],
   standalone: true
 })
-export class JobOverviewDrawerChartComponent implements OnInit, OnDestroy {
+export class JobOverviewDrawerChartComponent implements OnInit, OnDestroy, AfterViewInit {
   public data = [];
   public listOfMetricName: string[] = [];
   public listOfSelectedMetric: string[] = [];
   public listOfUnselectedMetric: string[] = [];
   public cacheMetricKey: string;
+  public dropdownWidth: number = 200; // Default width
```

Comment:
Isn't the current default width 300px? Can we keep the same current default width?